print-apps/oereb/toc.jrxml

```diff
@@ -268,14 +268,39 @@
 				<subreportExpression><![CDATA["general_info_and_disclaimer.jasper"]]></subreportExpression>
 			</subreport>
 			<subreport>
-				<reportElement stretchType="ContainerBottom" x="252" y="0" width="240" height="21" isPrintInFirstWholeBand="true" uuid="547dff1a-158f-43e9-a578-cd61513d0e62">
+				<reportElement stretchType="ContainerBottom" x="252" y="0" width="240" height="40" isPrintInFirstWholeBand="true" uuid="547dff1a-158f-43e9-a578-cd61513d0e62">
 					<property name="com.jaspersoft.studio.unit.width" value="px"/>
 					<property name="com.jaspersoft.studio.unit.height" value="px"/>
 				</reportElement>
+				<subreportParameter name="QRCodeRef">
+					<subreportParameterExpression><![CDATA[$P{QRCodeRef}]]></subreportParameterExpression>
+				</subreportParameter>
 				<dataSourceExpression><![CDATA[$P{DisclaimerDataSource}]]></dataSourceExpression>
 				<subreportExpression><![CDATA["disclaimer.jasper"]]></subreportExpression>
 			</subreport>
 		</band>
+		<band height="80" splitType="Stretch">
```

Comment:
Currently (with the test date) the QR code is moved to the next page. If we want to give it a try to avoid this I would suggest to add it in the same band as the disclaimer and the general info but add it in an other sub-report with the QRCodeLabel and the QRCodeRef.
Just note for me this is cosmetics and it might be overkill.

Reply:
@marionb thanks for the suggestion - I was thinking along the same lines when implementing it. 😃 I might adapt it this way, also depending on the comments from the others and possibly the confederation.

Reply:
Grouping the QR-code with other elements might be more appealing.